### ISSUE TITLE
Edge ordering

### ISSUE THREAD

**Issue by moret** [RisingStack/graffiti-mongoose#53]
Hi.

I was looking for a way to order edges based on the parent document array, but couldn't find a way. For example, given a model with `Person` and `Destinations` and a database:
```javascript
const destinations = [
  new Destination({name: 'London'},
  new Destination({name: 'Paris'},
  new Destination({name: 'Berlin'}
];

const people = [
  new Person({name: 'John', trips: [destinations[2], destinations[0]]}),
  new Person({name: 'Jane', trips: [destinations[1]]}),
  new Person({name: 'Jill', trips: [destinations[1], destinations[0]]})
];
```
and a query:
```graphql
{
  people {
    name
    trips {
      edges {
        node {
          name
        }
      }
    }
  }
}
```
I would expect the result:
```json
{
  "data": {
    "people": [
      {
        "name": "John",
        "trips": {
          "edges": [
            {
              "node": {
                "name": "Berlin"
              }
            },
            {
              "node": {
                "name": "London"
              }
            }
          ]
        }
      },
      {
        "name": "Jane",
        "trips": {
          "edges": [
            {
              "node": {
                "name": "Paris"
              }
            }
          ]
        }
      },
      {
        "name": "Jill",
        "trips": {
          "edges": [
            {
              "node": {
                "name": "Paris"
              }
            },
            {
              "node": {
                "name": "London"
              }
            }
          ]
        }
      },
    ]
  }
}
```
Is that currently possible with graffiti? I couldn't find a way to do it, nor to order in other collections. Thank you in advance for the attention!

**Comment by tothandras** [comment]
@moret It's not currently possible to specify an order. It could be implemented by adding an `orderBy` enum type to arguments based on the fields of the type. We can add this feature request to our list, or you can send in a PR ;). Also, you can always do the ordering on the client side.

**Comment by moret** [comment]
@tothandras I see. I would love to dive in and help with a PR, I'm just not sure if I will be able to right now. But I will keep an eye around. Thank you for your reply!

**Comment by tothandras** [comment]
@moret Please take a loot at #61!

**Comment by moret** [comment]
@tothandras looks great! In my use case I still need to use the association array own ordering which got solved in memory on the client as you suggested, but the `orderBy` solved all other sorting cases. Thanks a lot for the attention to that! 8)

**Comment by tothandras** [comment]
@moret Is it okay if I merge it as it is? :)

**Comment by moret** [comment]
@tothandras sure!

**Comment by tothandras** [comment]
@moret #61 is merger and released under `4.3.0`. ;)
